tests/Unit/Subscriber/Core/RegistratorSubscriberTest.php

```diff
@@ -104,6 +140,17 @@ public function testNodeFromRegistry()
         $this->subscriber->handleNodeFromRegistry($this->persistEvent->reveal());
     }
 
+    public function testReorderNodeFomRegistry()
```

Comment:
As already said I am against adding these comments, most of the time they are just duplicating information. The name already shows that it is testing if the node is set from the registry on the reorder event. If you think the name is not sufficient I would still prefer to change the name, instead of adding more comments.